src/openrct2/ride/Vehicle.cpp

```diff
@@ -7272,6 +7272,23 @@ void Vehicle::UpdateAnimationAnimalFlying()
         animationState = frameWaitTimes[animation_frame];
     }
 }
+/**
+ * Get the frame of animation for the current animationState based on animation speed and animation frames
+ */
+static uint8_t GetTargetFrame(rct_ride_entry_vehicle* vehicleEntry, uint32_t animationState)
```

Comment:
What does `animationState` mean? It's not very descriptive.

Reply:
It depends on the animation type. For most animation types, animationState is the sum of _vehicleVelocityF64E08. For others, it's a countup value. For one, it's a countdown value.

Reply:
The only animation types that call this number are the first. Since this was converted from a member function to a static function, I kept the original name.